data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -194,88 +194,84 @@
     field_names.ASTHMA_LOW_INCOME_FIELD,
-    field_names.DIABETES_FIELD,
-    field_names.DIABETES_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.ASTHMA_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.ASTHMA_FIELD,
     field_names.DIABETES_LOW_INCOME_FIELD,
-    field_names.HEART_DISEASE_FIELD,
-    field_names.HEART_DISEASE_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.DIABETES_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.DIABETES_FIELD,
     field_names.HEART_DISEASE_LOW_INCOME_FIELD,
-    field_names.LIFE_EXPECTANCY_FIELD,
-    field_names.LOW_LIFE_EXPECTANCY_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.HEART_DISEASE_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.HEART_DISEASE_FIELD,
     field_names.LOW_LIFE_EXPECTANCY_LOW_INCOME_FIELD,
-    field_names.MEDIAN_INCOME_AS_PERCENT_OF_AMI_FIELD,
+    field_names.LOW_LIFE_EXPECTANCY_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.LIFE_EXPECTANCY_FIELD,
+    field_names.LOW_MEDIAN_INCOME_LOW_HS_EDUCATION_FIELD,
     field_names.LOW_MEDIAN_INCOME_AS_PERCENT_OF_AMI_FIELD
     + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.LOW_MEDIAN_INCOME_LOW_HS_EDUCATION_FIELD,
-    field_names.LINGUISTIC_ISO_FIELD,
+    field_names.MEDIAN_INCOME_AS_PERCENT_OF_AMI_FIELD,
     field_names.LINGUISTIC_ISOLATION_LOW_HS_EDUCATION_FIELD,
-    field_names.UNEMPLOYMENT_FIELD,
-    field_names.UNEMPLOYMENT_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
     field_names.LINGUISTIC_ISO_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.POVERTY_LESS_THAN_100_FPL_FIELD,
+    field_names.LINGUISTIC_ISO_FIELD,
+    field_names.UNEMPLOYMENT_LOW_HS_EDUCATION_FIELD,
+    field_names.UNEMPLOYMENT_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.UNEMPLOYMENT_FIELD,
+    field_names.POVERTY_LOW_HS_EDUCATION_FIELD,
+    field_names.POVERTY_LESS_THAN_200_FPL_FIELD
+    + field_names.PERCENTILE_FIELD_SUFFIX,
     field_names.POVERTY_LESS_THAN_100_FPL_FIELD
     + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.POVERTY_LOW_HS_EDUCATION_FIELD,
-    field_names.HIGH_SCHOOL_ED_FIELD,
+    field_names.POVERTY_LESS_THAN_200_FPL_FIELD,
+    field_names.POVERTY_LESS_THAN_100_FPL_FIELD,
     field_names.HIGH_SCHOOL_ED_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.LOW_HS_EDUCATION_FIELD,
-    field_names.THRESHOLD_COUNT,
-    field_names.UNEMPLOYMENT_LOW_HS_EDUCATION_FIELD,
+    field_names.HIGH_SCHOOL_ED_FIELD,
     field_names.COMBINED_UNEMPLOYMENT_2010,
-    field_names.CENSUS_DECENNIAL_UNEMPLOYMENT_FIELD_2009,
     field_names.COMBINED_POVERTY_LESS_THAN_100_FPL_FIELD_2010,
     field_names.ISLAND_AREAS_UNEMPLOYMENT_LOW_HS_EDUCATION_FIELD,
     field_names.ISLAND_AREAS_POVERTY_LOW_HS_EDUCATION_FIELD,
     field_names.ISLAND_AREAS_LOW_MEDIAN_INCOME_LOW_HS_EDUCATION_FIELD,
-    field_names.ISLAND_AREAS_LOW_HS_EDUCATION_FIELD,
+    field_names.THRESHOLD_COUNT,
```

Comment:
I'm not exactly sure where this field is supposed to go, but I'm 90% sure it's supposed to be near the beginning of the column order.

Reply:
Done.